scripts/wrapper/withdraw.js

```diff
@@ -299,6 +299,11 @@ module.exports = function (web3, artifacts) {
         return str.split(",")
       },
     })
+    .option("executeOnchain", {
+      type: "boolean",
+      default: false,
+      describe: "Directly execute transaction on-chain instead of sending to the backend",
+    })
```

Comment:
This `argv` is also used in `transfer_funds_to_master.js`, can you also modify that file to make it consistent with the other withdrawing scripts?

Reply:
I'm not sure what you mean by this? All withdraw related scripts import the argvs from this file, so it should already be available for the transfer script.

Reply:
Sorry, I missed a file, all good!